vms/evm/sync/customrawdb/accessors_metadata_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package customrawdb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/rawdb"
+	"github.com/ava-labs/libevm/ethdb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTimeMarkers(t *testing.T) {
+	cases := []struct {
+		name   string
+		write  func(ethdb.KeyValueStore) error
+		read   func(ethdb.KeyValueStore) (time.Time, error)
+		delete func(ethdb.KeyValueStore) error
+	}{
+		{
+			name:   "offline_pruning",
+			read:   ReadOfflinePruning,
+			write:  WriteOfflinePruning,
+			delete: DeleteOfflinePruning,
+		},
+		{
+			name:   "populate_missing_tries",
+			read:   ReadPopulateMissingTries,
+			write:  WritePopulateMissingTries,
+			delete: DeletePopulateMissingTries,
+		},
```

Comment:
It looks like we're trying to avoid duplication, but I think the saying "duplication is cheaper than the wrong abstraction" is valid here. Inserting function calls into the test-table makes it difficult to reason about + this will be annoying to maintain if our api evolves (if the function signature deviates it becomes awkward). I think the duplication is pretty trivial - should we just split this across tests for each marker type?